clamscan/manager.c

```diff
@@ -1270,6 +1289,25 @@ int scanmanager(const struct optstruct *opts)
     } else {
         char *dbdir = freshdbdir();
 
+        if (optget(opts, "fail-if-cvd-older-than")->enabled) {
+            time_t cvd_age;
+            const long long days = optget(opts, "fail-if-cvd-older-than")->numarg;
+
+            if ((ret = cl_cvdgetage(opt->strarg, &cvd_age)) != CL_SUCCESS) {
+                logg(LOGG_ERROR, "%s\n", cl_strerror(ret));
+
+                ret = 2;
```

Comment:
I imagine you don't need to override `ret` here.

Reply:
I was just following the example below that line, with `cl_load()`, where it still gets set to `2` even though `ret` is non-zero. I don't know why that's been done, but I thought I shouldn't change behaviour. Of course I'm happy to remove the `ret = 2;` line if you think I should.
```c++
if ((ret = cl_load(opt->strarg, engine, &info.sigs, dboptions))) {
    logg(LOGG_ERROR, "%s\n", cl_strerror(ret));
                
    ret = 2;
    goto done;
}
```